checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleExecutor.java

```diff
@@ -92,25 +101,27 @@ private void executeWithClassLoader(URLClassLoader projectClassloader) {
             checker.addListener(listener);
 
             final File xmlReport = configuration.getTargetXmlReport();
-            if (xmlReport != null) {
+            if (Objects.nonNull(xmlReport)) {
```

Comment:
please move all refactoring for modern java in separate commit/PR , I will merge it quickly.